Map/Source/SquareMap.cpp

```diff
@@ -40,12 +41,19 @@ char SquareMap::printField(const Field& field)
 
 SquareMap::SquareMap(const int mapSize)
 {
-    for(int x = 0; x < mapSize; x++)
+    for(int x = -1; x <= mapSize; x++)
```

Comment:
co sie tutaj dzieje i czemu zaczynamy od -1 ?

Reply:
Zrobiłem wstawiany ścian i trzeba zacząć od tego że poza obszar mapSize, czyli na pozycji
 -1 wstawiamy Wall field oraz zmieniłem znak <= żeby znowu przejść o jedno pole dalej niż jest reszta mapy i wstawić tam Wall field. Wydaje mi się że jest to sensowne po przykładowo przy rozmiarze mapy 2 bez pozycji - 1i zmiany znaku na <= otrzymamy na mapie same ściany.

Reply:
ok, ale my nie chcemy miec ścian dookoła mapy. Tak ustaliliśmy. Te ściany będą tylko narysowane, nie będą cześcią mapy. Jedyne  ściany na mapie to ściany, które rozdzielają pokoje